src/components/Earn.jsx

```diff
@@ -179,7 +213,8 @@ export default function Earn({ currentWallet, makerRunning }) {
 
     if (isValid) {
       if (makerRunning === false) {
-        await startMakerService(feeAbs, feeRel, offertype, minsize)
+        const feeRelFactor = percentageToFactor(feeRel)
```

Comment:
Probably doesn't make a difference but I feel like it would be cleaner to do that conversion before setting the state. Basically keep the state a factor and keep the UI-related things on the UI layer. Meaning converting percent -> factor before setting the state and factor -> percent after reading the state before it is being rendered. Only if it's not a huge hassle though. What do you think—tell me if it doesn' t make sense.

Reply:
Way more comprehensible! Will do.